nevergrad/optimization/optimizerlib.py

```diff
@@ -753,6 +757,8 @@ def __init__(
 
             assert self.num_vars[i] >= 1, "At least one variable per optimizer."
             self.parametrizations += [p.Array(shape=(self.num_vars[i],))]
+            for param in self.parametrizations:
```

Comment:
This is were the problem was, the optimizers did not share the same random state

Reply:
Thx!